packages/web-components/fast-foundation/src/number-field/number-field.ts

```diff
@@ -247,6 +248,29 @@ export class NumberField extends FormAssociatedNumberField {
     public handleChange(): void {
         this.$emit("change");
     }
+
+    /**
+     * Handle keyboard events for the input.
+     *
+     * @param e - the keyboard event
+     * @internal
+     */
+    public handleKeyDown(e: KeyboardEvent): void | boolean {
+        if (e.defaultPrevented) {
+            return;
+        }
+
+        switch (e.keyCode) {
```

Comment:
We should be moving away from `.keyCode` since that's deprecated.  You should use `.key` instead as is done here:  https://github.com/microsoft/fast/blob/9bd00b78506786080d1033de7d6d06a797da3024/packages/web-components/fast-foundation/src/combobox/combobox.ts#L360

Reply:
```suggestion
        switch (e.key) {
```